content/community/contributing/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Contributing guide
+#linktitle: Videos
+weight: 10
+sidebar_multicard: false
+icon: callout-idea
+---
+ 
+
+We are welcoming new contributors to the Keptn project!
+
+## Getting started
+
+1️⃣ If you are new to Keptn and want to get started contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```

Comment:
Please don't use icons for numbering here. Go with: `1.` `2.`

Reply:
changed it to numbers